NotificationSystem/README.md

```diff
@@ -52,15 +52,15 @@ In the moderators flow:
 
 - A change in the Cosmos DB metadata store triggers the SendModeratorEmail function
 - If there is a newly detected orca call that requires a moderator to validate, the function fetches the relevant email list
-- The function then calls SendGrid to send emails to moderators
+- The function then calls AWS Simple Email Service to send emails to moderators
```

Comment:
What is the motivation for switching the service? Cost?

Reply:
Per my discussion with Scott, main motivation would be saving cost